bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -70,6 +71,10 @@ export default {
     query () {
       let direct, equipment, allPoints, points
       switch (this.type) {
+        case 'battery':
+          direct = findPoints(this.element.properties, 'Point_Status_LowBattery', true)
```

Comment:
I'm just realizing now that lines 75 & 76 here might even be unnecessary - these prioritize points directly under the Location over those under an Equipment... but having LowBattery points directly under a Location makes little sense I think, you would expect batteries to be under Equipment all the time?